### PR TITLE
create driver model and change get_report

### PR DESCRIPTION
**Pull request by zxc322**
- Add a Driver model and register it in admin.py
- Change app.libs.selenium_tools.get_report function ( replace `driver_maps` dict with drivers from database )
- To create drivers from deleted `drivers_maps` you can run `python3 manage.py runscript create_test_drivers` or do it manualy with  django-admin dashboard (script was created for tests) 
- as test I've saved output from command `python3 manage.py runscript weekly` to txt file in root of project before I made changes and after ( 2 txt files in root of project)

### CODE REVIEW

**Comment by SergeyKutsko** [comment]
@zxc322 good let's now add fleet model , so Uber bolt Uklon and new Fleet can be added to db and we can get report for new taxi fleet line On Taxi for example via Django admin

**Comment by zxc322** [comment]
> @zxc322 good let's now add fleet model , so Uber bolt Uklon and new Fleet can be added to db and we can get report for new taxi fleet line On Taxi for example via Django admin
I am confused a little bit: )
So what is our goal for now?  We want to save reports to db and bind it to Fleet model
```
class Fleet(models.Model):
    name = models.CharField()
```
```
class Report(models.Model)
    fleet = models.ForeignKey(Fleet, on_delete...)
    driver = models.ForeignKey(Driver, on_delete...)
    ... some report info...
```
So later we can make a query like:
```
SELECT * FROM Report WHERE Fleet=Uber (for example)
```
But I feel like I dont understand a real purpose of Fleet model. Can you please tell in more detail where we will use this model

**Comment by SergeyKutsko** [comment]
> > @zxc322 good let's now add fleet model , so Uber bolt Uklon and new Fleet can be added to db and we can get report for new taxi fleet line On Taxi for example via Django admin
> 
> I am confused a little bit: )
> 
> So what is our goal for now?  We want to save reports to db and bind it to Fleet model
> 
> ```
> 
> class Fleet(models.Model):
> 
>     name = models.CharField()
> 
> ```
> 
> ```
> 
> class Report(models.Model)
> 
>     fleet = models.ForeignKey(Fleet, on_delete...)
> 
>     driver = models.ForeignKey(Driver, on_delete...)
> 
>     ... some report info...
> 
> ```
> 
> So later we can make a query like:
> 
> ```
> 
> SELECT * FROM Report WHERE Fleet=Uber (for example)
> 
> ```
> 
> But I feel like I dont understand a real purpose of Fleet model. Can you please tell in more detail where we will use this model

Fleet is a company who receive payments from customers who use taxi services. It is also validate Drivers and Venicles(Cars). Fleet services like bolt Uklon Uber also taking their commissions before make payment to Fleet partner. We are fleet partner of all of this services.

**Comment by zxc322** [comment]
Alright, but I still dont understand where in our code we gonna use Fleet model.
Model `Driver` was created to replace static mapping from `get_report()` for example .
So  when I know why do we need this model and how we will use it ( or maybe what part of static code must be replaced with dinamyc) I can decide what fields in model do we need.
Sorry if my questions are stupid, but Im rly want to understand this task :)

**Comment by SergeyKutsko** [comment]
> Alright, but I still dont understand where in our code we gonna use Fleet model. Model `Driver` was created to replace static mapping from `get_report()` for example . So when I know why do we need this model and how we will use it ( or maybe what part of static code must be replaced with dinamyc) I can decide what fields in model do we need. Sorry if my questions are stupid, but Im rly want to understand this task :)

https://github.com/SergeyKutsko/universal_auto/issues/6

Take a look, description. Fleet will not directly use in calculations except its  name, fee exists just for information, because this fee already deducted from report by Fleet

**Comment by zxc322** [comment]
So what I did

- Add fleet model with only 1 field (fleet_name)
- Add model to django-admin
- Add field `fleet` as ForeignKey to `PaymentOrders` models and replase function `vendor()`  ( now we can return `self.fleet.fleet_name` instead
- You can create 3 Fleets by runing `python3 manage.py runscript create_test_fleets` or fo it manualy from django admin
- Also saved output from  script weekly report to `weekly_output_after_changes.txt`

File: scripts/create_test_fleets.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import sys
+
+sys.path.append('app/libs')
+from crete_test_data import create_fleet
+
+# you can run this script to add next fleets to database
+
+def run():
+    create_fleet(name='Uber', fees=0.65)
```
Comment (SergeyKutsko) [inline scripts/create_test_fleets.py:9]: Uber = 0.22
Bolt = 0.25
Uklon = 0.17

**Review by SergeyKutsko — CHANGES_REQUESTED** [review]
@zxc322 please resolve conflicts first.

**Pull request closed without merging**